server/tests/factories.js

```diff
@@ -0,0 +1,74 @@
+import Faker from 'faker';
+import User from '../models/user';
+import Device from '../models/device';
+import Group from '../models/group';
+import Ticket from '../models/ticket';
+import EscalationPolicy from '../models/escalationPolicy';
+
+const fixtures = {
+  user(user = {}) {
+    return {
+      id: user.id,
```

Comment:
No, I figured if you needed to explicitly define a uuid, you would, otherwise mongo would give it a default id.